pom.xml

```diff
@@ -19,6 +19,12 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
+    <parent>
+        <groupId>org.apache.camel</groupId>
+        <artifactId>camel-dependencies</artifactId>
+        <version>3.18.0</version>
```

Comment:
We released with 3.18.2 on 1.15.x

Reply:
Yep. As I wrote in the description, I left intentionally to test it. In fact, the checks proved that right :) - I'll add a commit to fix that and test as well.